mpc/src/honeybadger/mod.rs

```diff
@@ -88,6 +98,42 @@ pub enum HoneyBadgerError {
     ChannelClosed,
 }
 
+pub struct HoneyBadgerMPCClient<F: FftField, R: RBC> {
```

Comment:
I'm thinking there should be a Client trait that HoneyBadgeRMPCClient implements. This would allow us to switch between MPC protocols when we go to create the more "general" client which would have other forms of handling that are not specific to MPC.